config/routes.rb

```diff
@@ -10,4 +10,6 @@
                sessions: 'sessions',
                registrations: 'registrations'
              }
+  get '/books/:page', to: 'books#index'
+  get '/books/show/:id', to: 'books#show'
```

Comment:
```suggestion
  get '/books/:id', to: 'books#show'
```

you might want to change this route to match the apiary one here

https://wbooksapi.docs.apiary.io/#reference/books/member/information